tensorboard/plugins/scalar/tf_scalar_dashboard/tf-scalar-card.html

```diff
@@ -0,0 +1,288 @@
+<!--
+@license
+Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<link rel="import" href="../paper-item/paper-item.html">
+<link rel="import" href="../paper-dropdown-menu/paper-dropdown-menu.html">
+<link rel="import" href="../paper-icon-button/paper-icon-button.html">
+<link rel="import" href="../paper-menu/paper-menu.html">
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../tf-backend/tf-backend.html">
+<link rel="import" href="../vz-line-chart/vz-line-chart.html">
+<link rel="import" href="tf-scalar-chart.html">
+
+<!--
+  A card that contains an individual scalar chart.
+-->
+<dom-module id="tf-scalar-card">
+<template>
```

Comment:
It seems to me that `tf-scalar-card` should own the `tf-card-heading`. That is, the `tf-scalar-chart` component should display just a chart and nothing more: the log scale toggle, download links, etc. should be handled by the enclosing component.

Reply:
+1, done. It seems like cards could indeed differ in header logic (for instance, due to differences in merging descriptions).